src/lib/components/SearchBar/SearchBarILS.js

```diff
@@ -6,7 +6,8 @@ export class SearchBarILS extends Component {
   state = { currentValue: '' };
 
   componentDidMount() {
-    if (this.focusInput) {
+    const { autoFocus } = this.props;
+    if (autoFocus && this.focusInput) {
```

Comment:
I guess you did it, but did we use the `focusInput.focus()` anywhere else and maybe needs fix?

Reply:
I checked and it's only used in RSK but I already added the `autofocus={false}` in this cases

Reply:
does it still autofocus on desktop after this change ?